### ISSUE TITLE
create Sockets.createSocket not working

### ISSUE THREAD

**Issue by soufiansaidi** [ersinayaz/rn-samsung-tv-remote#2]
Whenever I run the application always return this error on the terminal

![Capture d’écran 2021-09-18 à 11 32 44](https://user-images.githubusercontent.com/11716618/133885876-2ff3b84f-18c4-410e-aea8-b86020c86850.png)

**Comment by ersinayaz** [comment]
Hi,
Could you please share the line of code in detail so that I can understand the problem?

**Comment by soufiansaidi** [comment]
This is my component

`import React, { useEffect, useState } from 'react'
import { SamsungRemote, DeviceDiscovery } from '@ersinayaz/rn-samsung-tv-remote';
import { Alert, Text, View } from 'react-native';


const Remote = () => {
    const [device, setDevice] = useState({})
    useEffect(() => {
        handleGetDevices()
    }, [])

    const handleGetDevices = async () => {
        Alert.alert('Start handleGetDevices')
        const devices = await DeviceDiscovery.discover("urn:samsung.com:device:RemoteControlReceiver:1");
        Alert.alert('after DeviceDiscovery')
        setDevice(devices)
        Alert.alert(JSON.stringify(devices))
        Alert.alert('end handleGetDevices')
    }

    return (
        <View>
            <Text>Remote</Text>
            <Text>{JSON.stringify(device)}</Text>
        </View>
    )
}

export default Remote
`

**Comment by ersinayaz** [comment]
Can you try installing the following dependencies?

```
yarn add react-native-udp@3.1.0 events@3.3.0 buffer@5.6.0
npx pod-install
```

restart project.

**Comment by ersinayaz** [comment]
Hi, are you still having problems?

**Comment by soufiansaidi** [comment]
@soufiansaidi yes Problem solved after I install react-native-udp@3.1.0 events@3.3.0 buffer@5.6.0
